modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -64,53 +70,67 @@ http://www.mulesoft.org/schema/mule/ee/tracking http://www.mulesoft.org/schema/m
 ----
 =====
 
-Because CloudHub synchronizes your schedules automatically, overriding any synchronize property set on the flows containing the polling scopes, there is no need to do any other special configuration.
+CloudHub synchronizes your schedules automatically, overriding any synchronize property set on the flows containing the polling scopes, so you don't need to do any other special configuration.
```

Comment:
```suggestion
CloudHub synchronizes your schedules automatically, overriding any synchronize property set on the flows containing the polling scopes so that you don't need to perform any other special configuration.
```